misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -73,6 +73,9 @@ public List<String> skipList() {
                 "qualified_identifier_assert_08.bal",
                 "conditional_expr_source_28.bal",
                 "resiliency_source_04.bal",
+                "record_type_def_source_14.bal",
+                "object_type_def_source_12.bal",
+                "anon_func_source_01.bal",
```

Comment:
In order to update parser test cases with correct formatting we can use the following steps:
- Change the formatting in the required parser test cases.
- Change the `UPDATE_ASSERTS` variable to `true`. https://github.com/ballerina-platform/ballerina-lang/blob/01f06c390aa875e0cc2134a74f8754d0c045e193/compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/ParserTestUtils.java#L75
- Run the parser test cases.
-  Change the `UPDATE_ASSERTS` variable back to `false`. 
- Commit the changed `.json` and `.bal` resource files.

When changing the parser test cases, make sure to get the PR approved by a parser team member as well.